dashboard/app/models/user.rb

```diff
@@ -524,7 +524,7 @@ def sections
   validates :name, length: {within: 1..70}, allow_blank: true
   validates :name, no_utf8mb4: true
 
-  defer_age = proc {|user| %w(google_oauth2 clever powerschool).include?(user.provider) || user.sponsored?}
+  defer_age = proc {|user| %w(google_oauth2 clever powerschool migrated).include?(user.provider) || user.sponsored?}
```

Comment:
@daynew there wasn't a TODO comment here, but when we added this I assume it wasn't intended to be permanent, correct?

Reply:
Oh yes, you are correct. oops! Nice catch :) I'm pretty sure we need to figure out a proper solution for defering age. We probably need to add something like `|| lti?`